lib/http/endpoint.js

```diff
@@ -192,20 +192,28 @@ const endpointBase = ({ preprocessor = noop, before = noop, resultWriter, errorW
 // ENDPOINT FORMAT SPECIALIZATIONS
 
 ////////////////////////////////////////
-// JSON/default
 
 const streamErrorHandler = (response) => () => {
   response.addTrailers({ Status: 'Error' }); // TODO: improve response content.
 };
-const defaultResultWriter = (result, request, response, next) => {
-  if (!response.hasHeader('Content-Type')) response.type('json');
-
+const pipelineResult = (result, response, next) => {
   if (result instanceof PartialPipe) {
     result.streams.at(-1).on('error', streamErrorHandler(response));
     result.with(response).pipeline((err) => next?.(err));
+    return true;
   } else if (result.pipe != null) {
     result.on('error', streamErrorHandler(response));
     pipeline(result, response, (err) => err && next?.(err));
+    return true;
+  }
+};
```

Comment:
I know `undefined` is falsy, but I think it'd be nice if the function returned `false` if it doesn't return `true`.

Reply:
Added.